generator/parser/tokens.cpp

```diff
@@ -152,7 +152,9 @@ static char const * const _S_token_names[] = {
   "xor",
   "xor_eq",
   "Q_ENUMS",
-  "Q_ENUM"
+  "Q_ENUM",
+  "Q_INVOKABLE",
+  "<end>"
```

Comment:
Is the `<end>` actually needed, or just there as a sentinel?

Reply:
Just there as a sentinel.

Reply:
...and gone again after the rebase.